spec/unit/resources/logical_interconnect/bulk_inconsistency_validate_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative './../../../spec_helper'
+
+describe 'oneview_test_api2000_synergy::logical_interconnect_bulk_inconsistency_validate' do
```

Comment:
you have mentioned cookbook name as "oneview_test_api2000_c7000". So mention the same here.